.github/workflows/release.yml

```diff
@@ -0,0 +1,53 @@
+name: Node CI
+on:
+  push:
+    branches:
+      - main
+  pull_request_target:
```

Comment:
Are we sure we want to allow npm releases from forks? Doesn't `pull_request_target` allow an external contributor to release a canary package with any code they like in it. 🤔